flake.nix

```diff
@@ -0,0 +1,22 @@
+{
+  description = "Home Manager for Nix";
```

Comment:
@bqv could you add a usage example to the README?

Reply:
Honestly, a usage example would require an entire flake.nix for a NixOS system (I'm unsure how one would use this on non-NixOS for just home-manager support). Maybe it would be better to point to examples of home-manager being used as a flake, rather than showing a code snippet (which would get quite large).

For reference, here are current examples of using home-manager as a flake for NixOS (I'm not aware of any more):

* https://github.com/bqv/nixrc/blob/d65c767bace3cc057e6aec2a7202a131d234be58/flake.nix#L237
* https://github.com/cole-h/nixos-config/blob/8f797361d8956b4e1696ed4f1088aa25999b2aa2/flake.nix#L91

Reply:
Nonetheless, I think it might be helpful to have something - I've co-opted your config into something vaguely demonstrative :) #1468